src/BenchmarksApps/TodosApi/TodosApi.json

```diff
@@ -201,23 +239,31 @@
             "name": "title",
             "in": "query",
             "required": true,
-            "style": "form",
```

Comment:
`"form"` is the default style for parameters in the query so there's no need to set it explicitly.